docs/spec/facets/facets.md

```diff
@@ -94,112 +94,3 @@ Here is an example of the four facets in action. Notice the element `facets` und
 }
 ```
 For more information of what kind of facets are available as part of OpenLineage spec, pleaes refer to the sub sections `Run Facets`, `Job Facets`, and `Dataset Facets` of this document.
-
-## Custom Facets
```

Comment:
Just making sure - do we intend to remove this content on the facets/facets.md page?

Reply:
No, hmm..